flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -255,6 +255,20 @@ public void testCreateTable() throws Exception {
         tableEnv.executeSql("create table if not exists tbl5 (m map<bigint,string>)");
         hiveTable = hiveCatalog.getHiveTable(new ObjectPath("default", "tbl5"));
         assertEquals(createdTimeForTableExists, hiveTable.getCreateTime());
+
+        // test describe table
+        assertEquals(
+                3,
+                CollectionUtil.iteratorToList(tableEnv.executeSql("desc tbl1").collect()).size());
+        assertEquals(
+                1,
+                CollectionUtil.iteratorToList(tableEnv.executeSql("describe tbl2").collect())
+                        .size());
+        assertEquals(
+                3,
+                CollectionUtil.iteratorToList(
```

Comment:
Why not comapre the describe result?

Reply:
Because hive dialect doesn't control the describe result and the planner is free to change it in the future. Maybe I can assert the parsed operation is a `DescribeTableOperation`. WDYT?

Reply:
For Flink dialect, we test the describe result in `flink-table/flink-sql-client/src/test/resources/sql/table.q`. 
Is it possible to test this feature in `flink-connectors/flink-connector-hive/src/test/resources/query-test`? 
If this needs some refactoring, I'm fine with merging this fix first to not block the release, and we can do that in a follow-up PR. 

Maybe we can move hive dialect tests to sql-client to reuse the script testing framework.

Reply:
We can compare the result in this test. But my point is it's `TableEnvironmentImpl` that decides what's the result looks like. Verifying something hive dialect can't control in a hive dialect test doesn't make much sense. From hive dialect perspective, anything returned from `TableEnvironmentImpl` would be considered legitimate.

Reply:
Yes. I just wondering we don't have an end-to-end test for these statements. For example, what if we bridge the `desc t1` to a `DescribeTableOperation` with a wrong `extended` flag, what if `TableEnvironmentImpl` doesn't implement `extended` correctly (it seems we didn't support `extended` yet in `TableEnvironment`). 

I'm fine with just verifying parsed operation here, but would be better to have e2e tests in the near future to cover more code paths.